### PR TITLE
add xmerl-1.3.32

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydin <batuhan.apaydin@chainguard.dev>

Related:
- https://github.com/chainguard-dev/image-requests/issues/102
- https://pkgs.alpinelinux.org/contents?file=&path=&name=erlang&branch=edge&repo=community&arch=x86

Screenshot 2023-06-24 at 19 45 08

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [x] REQUIRED - The package is added to `packages.txt`

#### For security-related PRs

- [ ] The security fix is recorded in `advisories` and `secfixes`

#### For version bump PRs

- [ ] The `epoch` field is reset to 0
- [ ] Patch source: _patch source here_

### CODE REVIEW

**Review by kaniini — CHANGES_REQUESTED** [review]
The subpackage names need to describe their relationship to erlang.   Also the versions of the erlang applications should probably be omitted from the package name.

**Comment by developer-guy** [comment]
> The subpackage names need to describe their relationship to erlang. 

So, we should add smth like "erlang-" prefix on front of them.

> Also the versions of the erlang applications should probably be omitted from the package name.

As we have erlang-25 (25.3.2.2-r0) and erlang-26 (26.0.1-r0) packages, so, I believe, they have both different version of the sub-packages, for example, for erlang-26 the sub-package xmerl* version is 1.3.2 but for erlang-25, it is 1.3.31.1, so, in that situtation, what should we do if we shouldn't provide the version information in the sub-package name? Thanks

**Comment by kaniini** [comment]
We should use erlang-26-xmerl etc.

**Review by imjasonh — COMMENTED** [review]
This change lgtm but I'd like to wait for @kaniini's approval since she had comments.

File: erlang-26.yaml
Comment (kaniini) [inline erlang-26.yaml:67]: Why not use range here?

**Review by kaniini — DISMISSED** [review]
Please flatten and use range instead of repeating the same block over and over again.

File: erlang-26.yaml
Comment (developer-guy) [reply erlang-26.yaml:67]: great suggestion!

**Review by imjasonh — APPROVED** [review]

**Pull request merged**
